ldap/servers/slapd/conntable.c

```diff
@@ -122,7 +122,8 @@ connection_table_new(int table_size)
     ct->list_num = config_get_num_listeners();
     ct->num_active = (int *)slapi_ch_calloc(1, ct->list_num * sizeof(int));
     ct->size = table_size - (table_size % ct->list_num);
-    ct->list_size = ct->size/ct->list_num;
+    /* Account for first slot of each list being used as head (c_next).  */
+    ct->list_size = (ct->size/ct->list_num)+1;
```

Comment:
There was a bug when we were mapping the connection table lists to the freelist, as slot 0 of each ct list is used as head of the linked list it was skipped,  resulting in each list being 1 less than it should be. 

This became apparent when the freelist offset is used to determine if the server has room for a new connection.